### ISSUE TITLE
Should locale be a select instead of a text input?

### ISSUE THREAD

**Issue by psrpinto** [WordPress/wporg-gp-translation-events#19]
Currently the Locale form input is a text field and the user can input any text, e.g `foo`.

I'm wondering, should it be a select instead, with only the locales that exist in GlotPress db?

**Comment by trymebytes** [comment]
Yea, I agree that it should be a select dropdown. @akirk and I had a discussion about this and we agreed to remove/hide this locale field for now. So, after the initial release, we can continue working on it.

**Comment by amieiro** [comment]
Yes, we should have a multi selection dropdown, so the user can select more than one language for the event.

**Comment by obenland** [comment]
Closing as this repository is being archived. Translation Events is now maintained in the WordPress.org Meta repository (https://meta.trac.wordpress.org/ticket/8368). If this is still relevant, please open a Meta Trac ticket under "Translate Site & Plugins" (https://meta.trac.wordpress.org/newticket?component=Translate+Site+%26+Plugins), or a PR against https://github.com/WordPress/wordpress.org.
